LBHackney-IT/lbh-social-care
# Align pages

**What**  
Maintainance PR :)
- Aligned pages file extension
- Upgraded dependencies to the latest version (the dev dependencies were ported but not upgraded, my bad 😓)
- Added missing optional dependencies, now yarn is not complaining anymore 
- Prettified all the things (aligned to version 2 of prettier, I love the "python comma" 😅)

**Why**  
The code is more standardized :)

**How to test it**
- checkout this branch
- install the dependencies
- everything should work as before
